pymc4/distributions/discrete.py

```diff
@@ -176,11 +177,13 @@ class BetaBinomial(BoundedDiscreteDistribution):
     """
 
     def __init__(self, name, total_count, concentration0, concentration1, **kwargs):
+        concentration0 = np.array(concentration0)
+        concentration1 = np.array(concentration1)
         super().__init__(
             name,
             total_count=total_count,
-            concentration0=concentration0,
-            concentration1=concentration1,
+            concentration0=concentration0.reshape(concentration0.shape + (1,)),
+            concentration1=concentration1.reshape(concentration1.shape + (1,)),
```

Comment:
Never reshape under the hood while you instantiate a distribution. This will bring many shape issues down the line